docs/tutorial/forge-overview.md

```diff
@@ -0,0 +1,30 @@
+# Distributing Apps With Electron Forge
+
+Electron Forge is a complete tool for creating, publishing, and installing modern Electron applications. Electron Forge unifies the existing (and well maintained) build tools for Electron development into a simple, easy to use package so
+that anyone can jump right in to Electron development.
```

Comment:
```suggestion
Electron Forge is a complete tool for creating, publishing, and installing modern Electron applications. Electron Forge unifies Electron's build tooling ecosystem into a single extensible interface so
that anyone can jump right in to Electron development.
```

Some copy edits:
* Didn't want to confuse `build tools` with `@electron/build-tools`
* `(and well maintained)` is a bit unnecessary 😂 
* Forge is not a single package, it's a monorepo with dozens of them!